src/assets/styles/layout.scss

```diff
@@ -0,0 +1,47 @@
+html,
```

Comment:
Added some basic layout stuff cause I needed it to style the desktop-ish size event list. So this is basically a header, then a primary area beneath it that fulls up the rest of the screen, and map on the left list on the right, with each of them also filling up the entire height.

I used a little flexbox but I only know it well enough to be dangerous, it might be terrible :grin: Absolutely positioned columns inside a flex container.....probably there is a better way but this is not the primary focus of this PR. Might be a good idea to address the layout separately if this is too janky. If anyone has thoughts LMK!